docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md

```diff
@@ -1,100 +1,132 @@
+ms.date: 09/17/2021
 ms.prod: excel
 ms.localizationpriority: high
 ---
 
 # Tutorial: Share data and events between Excel custom functions and the task pane
 
-You can configure your Excel add-in to use a shared runtime. This makes it possible to shared global data, or send events between the task pane and custom functions.
+You can configure your Excel add-in to use a shared runtime. This makes it possible to shared global data, or send events between the task pane and custom functions. For most custom functions scenarios, we recommend using a shared runtime, unless you have a specific reason to use a non-task pane (UI-less) custom function. This tutorial assumes you're familiar with using the Yo Office generator to create add-in projects. Consider completing the [Excel custom functions tutorial](excel-tutorial-create-custom-functions.md), if you haven't already.
 
-For most custom functions scenarios, we recommend using a shared runtime, unless you have a specific reason to use a non-task pane (UI-less) custom function.
+## Create the add-in project
 
-This tutorial assumes you're familiar with using the Yo Office generator to create add-in projects. Consider completing the [Excel custom functions tutorial](excel-tutorial-create-custom-functions.md), if you haven't already.
+You'll use the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) to create the Excel add-in project.
 
-## Create the add-in project
+- To generate an Excel add-in with custom functions, run the command `yo office --projectType excel-functions --name 'Excel shared runtime add-in' --host excel --js true`.
 
-Use the Yeoman generator to create an Excel add-in project. Run the following command and then answer the prompts with the following answers.
+The generator will create the project and install supporting Node components.
 
-```command line
-yo office
-```
+## Configure the manifest
 
-- Choose a project type: **Excel Custom Functions Add-in project**
-- Choose a script type: **JavaScript**
-- What do you want to name your add-in? **My Office Add-in**
+Follow these steps to configure the add-in project to use a shared runtime.
 
-![Screenshot showing the prompts and answers for the Yeoman generator in a command line interface.](../images/yo-office-excel-project.png)
+1. Start Visual Studio Code and open the add-in project you generated.
+1. Open the **manifest.xml** file.
+1. Update the requirements section to use the [shared runtime](../reference/requirement-sets/shared-runtime-requirement-sets.md) instead of the custom function runtime. The XML should appear as follows.
 
-After you complete the wizard, the generator creates the project and installs supporting Node components.
+    ```xml
+    <Hosts>
+      <Host Name="Workbook"/>
+    </Hosts>
+    <Requirements>
+      <Sets DefaultMinVersion="1.1">
+        <Set Name="SharedRuntime" MinVersion="1.1"/>
+      </Sets>
+    </Requirements>
+    <DefaultSettings>
+    ```
```

Comment:
I like having the context for where Requirements is in the file, but I don't like having to copy everything when only the one part is getting replaced.

Reply:
It's tricky because over time we've had issues where people get confused about where to put some of the XML, or accidentally put it inside another tag and such. So I've added some context about where it is located to try to help. But yeah, then you can't just copy paste...